app/api/request/route.ts

```diff
@@ -8,8 +8,8 @@ import { devnetFaucetURL, ticker } from "@/lib/constants";
 export const POST = async (req: Request) => {
   try {
     const { walletAddress, amount } = await req.json();
-    if (amount > 250000) {
-      return new Response(JSON.stringify({ error: `You can only request up to 0.25 ${ticker} per request.` }), { status: 400 });
+    if (amount > 2000000000) {
```

Comment:
For large numbers with more than three zeros, we can use underscores `2_000_000_000 ` for readability.